### ISSUE TITLE
I want to dial it back

### ISSUE THREAD

**Issue by cdils** [cdils/utility-pro#120]
After much thought, I've decided to scale back on the changes to UP 2.0. I take full ownership as I told @GaryJones to take the theme down its current path. That said, we're creating a product that is too many leaps ahead of the current market which will result in a difficult learning curve for users and a significantly increased support burden.

I'm proposing the following as part of a 2.0 release:

- Refreshed color scheme with new color options (that all meet WCAG2.0 AA accessibility guidelines)
- Add style support for Easy Digital Downloads & WooCommerce
- Add editor styles
- Gulp workflow
- Susy (instead of Bourbon/Neat)
- Composer 

In other words, move forward with front-end changes already in motion and drop the object-oriented approach. I would like to keep it PHP 7+.

@GaryJones You've put in significant work to refactor the theme to an object-oriented approach. If you're interested in forking the current UP 2.0 dev branch, renaming it, and selling it (or giving it away), I am okay with that. I don't want you to lose your efforts there.

Thoughts?
